src/libraries/System.Runtime.Extensions/tests/System/Environment.StackTrace.cs

```diff
@@ -15,8 +16,9 @@ public class EnvironmentStackTrace
         static string s_stackTrace;
 
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/73051", typeof(PlatformDetection), nameof(PlatformDetection.IsNativeAot))]
         [ActiveIssue("https://github.com/mono/mono/issues/15315", TestRuntimes.Mono)]
+        // Retain parameter names for NativeAOT
+        [DynamicDependency(DynamicallyAccessedMemberTypes.NonPublicMethods, typeof(EnvironmentStackTrace))]
```

Comment:
Is this needed?

We should be keeping the test assemblies fully - without that, the `[Fact]` methods would disappear as well because xunit is reflection-locating them.